Grids AI Assistant: Add (Update) API Descriptions Part 3 (#8815)
DevExpress/devextreme-documentation: api-reference/40 Common Types/15 grids/FilterExprTree/FilterExprTree.md

---
id: FilterExprTree
module: common/grids
export: FilterExprTree
---
---
##### shortDescription
A complex filter expression in a flat data structure.

---

> **FilterExprTree** stores a filter expression tree in a flat list instead of a hierarchical structure. A flat data structure ensures broader AI service compatibility for the `filterValue` AI Assistant command.

Reviewer: ```suggestion
**FilterExprTree** stores a filter expression tree as a flat list rather than a hierarchical structure. This flat structure improves compatibility with AI services for the AI Assistant's `filterValue` command.
```
